appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -413,13 +413,10 @@
 ;; Commented out -- we only send reports from the setting menu choice
 ;;         (com.google.appinventor.components.runtime.ReplApplication:reportError ex)
          (if isrepl
-             (when ((this):toastAllowed)
```

Comment:
There is a big shift in functionality here that I am not sure that I agree with. Note that in the current version of the code, the toast is shown in the companion (isrepl is true) but in compiled apps the runtime error alert is shown (isrepl is false). The code in the PR has been changed to show the runtime error alert when in the companion (isrepl is true) and do nothing in the case of compiled apps. I do not believe this is the intended effect. Also, the runtime error alert is a terminal state and the only option at that point is to end the app. This is not a behavior we want in the repl.

Reply:
I've fixed it, please rereview